en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -461,11 +482,11 @@ paths:
             type: string
       responses:
         '200':
-          description: Action is deactivated
+          description: Action Deactivated
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ActionBasicResponseInactive'
+                $ref: '#/components/schemas/ActionBasicResponse'
```

Comment:
By adding this it will show 'ACTIVE' as the status in the api response. Shall we add an example response for this if we are going with one model ?

Reply:
Well the API for retrieving all actions by action type will have the sample.

Can you elaborate on what do you mean by that this will show 'ACTIVE' ?
Why was that intentionally removed before ?

Reply:
This is be the action deactivate api. So the api response of this API should always show 'INACTIVE' as the action status. But using the same model used for the activate-action api response in here will show the example values added in that model ('ACTIVE' for status)

Reply:
Noted.

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4969/commits/54381489909728526076210e505bcdac5e2dcd26